.travis.yml

```diff
@@ -23,7 +23,7 @@ run:
 
 
 script:
-  - coverage run --source='.' manage.py test 
+  - coverage run --source='.' manage.py test --settings=authors.settings.testing
 
 after_success:
   - coveralls
```

Comment:
Get rid of the red arrows when you see them

Reply:
I have removed them.